docs/addons/kubecost.md

```diff
@@ -0,0 +1,77 @@
+# Kubecost AddOn
+
+[Kubecost](https://kubecost.com) provides real-time cost visibility and insights by uncovering patterns that create overspending on infrastructure to help teams prioritize where to focus optimization efforts. By identifying root causes for negative patterns, customers using Kubecost save 30-50% or more of their Kubernetes cloud infrastructure costs. To read more about Kubecost and how to use it, see the [product and technical docs](https://docs.kubecost.com/getting-started).
+
+## Installation
+
+Using [npm](https://npmjs.org):
+
+```sh
+$ npm install @kubecost/kubecost-ssp-addon
+```
+
+## Usage
+
+```typescript
+import * as cdk from '@aws-cdk/core';
+import { EksBlueprint } from '@aws-quickstart/ssp-amazon-eks';
+import { KubecostAddOn } from '@kubecost/kubecost-ssp-addon';
+
+
+export default class KubecostConstruct extends cdk.Construct {
+    constructor(scope: cdk.Construct, id: string) {
+        super(scope, id);
+        // AddOns for the cluster
+        // Include the Kubecost AddOn in your list
+        // Grab your token from kubecost.com/install (it's free)
+        const stackId = `${id}-blueprint`;
+
+        EksBlueprint.builder()
+            .account(process.env.CDK_DEFAULT_ACCOUNT!)
+            .region(process.env.CDK_DEFAULT_REGION)
+            .addOns(new KubecostAddOn())
+            .build(scope, stackId);
+    }
+}
+```
+
+## `KubecostAddOn` Options (props)
+
+#### `namespace: string` (optional)
+
+The namespace where Kubecost will be installed. Defaults to `kubecost`.
+
+#### `kubecostToken: string` (optional)
```

Comment:
Some customers may object to token being part of their source code. Let's think of a solution that leverages CSI Secrets Store driver or any other Kubernetes native approach (including straight k8s secrets) to pass it over.

Reply:
I've removed it from the docs, but kept the functionality in — it is an optional parameter.

Reply:
Is it a token that is sensitive in nature as far repercussions of falling into the wrong hands?
I can provide an approach how to leverage the CSI driver and AWS Secret manager for integration shortly if it helps. It creates a very visible precedent and others will be tempted to use the same approach for credentials, etc.
Do you think your customers will be open to use it by passing directly in the code?